test/asynchronous/test_collection.py

```diff
@@ -40,21 +40,17 @@
     wait_until,
 )
 
-from bson import encode
+from bson import RawBSONDocument, encode
```

Comment:
Does test/async ever get run in CI?

Reply:
No, left in as an example for our future test asynchronization work